ckanext/benap/validators.py

```diff
@@ -52,3 +52,23 @@ def is_after_start(key, flattened_data, errors, context):
                 raise Invalid(_('Start date must be before end date'))
     return True
 
+https_pattern = re.compile(
+    r"^https:\/\/"
+)
+
+http_pattern = re.compile(
+    r"^http:\/\/"
+)
```

Comment:
patterns bovenaan plaatsen bij ander pattern (phone line 10) om overzichtelijk te houden

Reply:
done in commit 099bf257